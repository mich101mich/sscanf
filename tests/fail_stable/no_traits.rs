fn main() {
    sscanf::scanf!("hi", "{}", std::vec::Vec<usize>);
}
