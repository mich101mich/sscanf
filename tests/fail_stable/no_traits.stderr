error[E0277]: the trait bound `Vec<usize>: RegexRepresentation` is not satisfied
 --> $DIR/no_traits.rs:2:32
  |
2 |     sscanf::scanf!("hi", "{}", std::vec::Vec<usize>);
  |                                ^^^^^^^^^^^^^^^^^^^^ the trait `RegexRepresentation` is not implemented for `Vec<usize>`
  |
  = note: required by `sscanf::RegexRepresentation::REGEX`
